2023-10-04T00:22:51.5186713Z ##[group]Run actions/setup-python@v4
2023-10-04T00:22:51.5186971Z with:
2023-10-04T00:22:51.5187180Z   python-version: 3.8
2023-10-04T00:22:51.5187427Z   architecture: x64
2023-10-04T00:22:51.5187628Z   cache: pip
2023-10-04T00:22:51.5187835Z   check-latest: false
2023-10-04T00:22:51.5188170Z   token: ***
2023-10-04T00:22:51.5188392Z   update-environment: true
2023-10-04T00:22:51.5188634Z   allow-prereleases: false
2023-10-04T00:22:51.5188856Z ##[endgroup]
2023-10-04T00:22:51.6939238Z ##[group]Installed versions
2023-10-04T00:22:51.7023761Z Successfully set up CPython (3.8.18)
2023-10-04T00:22:51.7024666Z ##[endgroup]
2023-10-04T00:22:52.5082909Z [command]/opt/hostedtoolcache/Python/3.8.18/x64/bin/pip cache dir
2023-10-04T00:22:54.9981125Z /home/runner/.cache/pip
2023-10-04T00:22:56.5346732Z Received 6245959 of 6245959 (100.0%), 6.5 MBs/sec
2023-10-04T00:22:56.5347584Z Cache Size: ~6 MB (6245959 B)
2023-10-04T00:22:56.5377285Z [command]/usr/bin/tar -xf /home/runner/work/_temp/d7121ea2-482e-43a9-9c7a-8fcf91b9df40/cache.tzst -P -C /home/runner/work/GitHub_Stats_A/GitHub_Stats_A --use-compress-program unzstd
2023-10-04T00:22:56.5707531Z Cache restored successfully
2023-10-04T00:22:56.5724981Z Cache restored from key: setup-python-Linux-22.04-Ubuntu-python-3.8.18-pip-60f2a6c0212a9b74a155cd81b58dd07fd8b180833132566260d95f4de42fe218
